zipkin-server/README.md

```diff
@@ -110,6 +110,7 @@ zipkin-server is a drop-in replacement for the [scala query service](https://git
 [yaml configuration](src/main/resources/zipkin-server-shared.yml) binds the following environment variables from zipkin-scala:
 
     * `QUERY_PORT`: Listen port for the http api and web ui; Defaults to 9411
+    * `QUERY_ENABLED`: False disables the query api and UI assets; Defaults to true
```

Comment:
If we do remove the `havingValue` parameter on the `@ConditionalOnProperty` sites, perhaps writing `false` (formatted) makes it clearer we're looking for that specifically to disable querying. With the `havingValue` anything but `true` will disable, I believe.